app/templates/views/check/column-errors.html

```diff
@@ -10,7 +10,12 @@
 {% endblock %}
 
 {% block backLink %}
-{{ usaBackLink({ "href": back_link }) }}
+<!--hide back link for one-off sends because the user never created a csv file-->
+{% if recipients.__len__() == 1 and not recipients.allowed_to_send_to and not recipients.missing_column_headers %}
```

Comment:
When using methods with double underscores on an object, you're tapping into the internal implementation details of an object and assuming the method is implemented, which may not always be the case!

In this case, would using the global `len()` method work instead?  This has fallback conditions internally and proper error handling should a class not implement the `__len__()` method on it:

```
{% if len(recipients) == 1 and not recipients.allowed_to_send_to and not recipients.missing_column_headers %}
```

My guess is this doesn't work though, right?  If that's the case, Jinja has a [`length()` filter](https://jinja.palletsprojects.com/en/3.1.x/templates/#jinja-filters.length) for this purpose, so this might be what we're looking for:

```suggestion
{% if recipients|length == 1 and not recipients.allowed_to_send_to and not recipients.missing_column_headers %}
```

Reply:
Yeah, len() didn't work.  Will try the jinja way